src/assets/styles/_colors.scss

```diff
@@ -0,0 +1,44 @@
+:root {
```

Comment:
@oknoway not sure if you did your mapping for converting to calcite-colors 2.0, if not this may be helpful